replication/binlogsyncer.go

```diff
@@ -496,6 +498,12 @@ func (b *BinlogSyncer) writeBinlogDumpMariadbGTIDCommand(gset GTIDSet) error {
 func (b *BinlogSyncer) localHostname() string {
 	if len(b.cfg.Localhost) == 0 {
 		h, _ := os.Hostname()
+		// if os.Hostname() lengther than 60, use md5sum instead
+		if len(h) > 60 {
```

Comment:
why md5? 

seem this change is not relevant to this PR.

Reply:
Current situation is that my company's  go-mysql-elasticsearch runs in K8S, the hostname set by our deliver system is longer than 62. And it's not easy to add an cfg.localhost for go-mysql-elasticsearch. I can remove these code  in this PR.
Is there suggestion for this kind of situation like K8S ,hostname is very long.